### PR TITLE
1.6.4 Data

### PR DESCRIPTION
**Pull request by halotroop2288**
I'm planning on going back to 1.5.2 for a personal project, and I figured I would start with this.
This PR is probably not properly done, and I would appreciate some pointers on finishing it.

### CODE REVIEW

File: data/pc/1.8/enchantments.json [hunk 1]
```diff
@@ -514,47 +514,5 @@
     "curse": false,
     "tradeable": true,
     "discoverable": true
-  },
```
Comment (halotroop2288) [inline data/pc/1.8/enchantments.json:517]: oops. TODO

File: data/pc/1.8/recipes.json [hunk 2]
```diff
@@ -203,51 +203,6 @@
         "id": 5,
         "metadata": 2
       }
-    },
```
Comment (halotroop2288) [inline data/pc/1.8/recipes.json:206]: ditto

File: data/pc/1.6/protocol.json [hunk 3]
```diff
@@ -0,0 +1,2898 @@
+{
```
Comment (halotroop2288) [inline data/pc/1.6/protocol.json:1]: TODO: Protocol has not been touched after copying from 1.7, so some things will inevitably be wrong.

**Comment by nickelpro** [comment]
Protocol docs are here:
https://wiki.vg/index.php?title=Protocol&oldid=4899

**Comment by rom1504** [comment]
hi,
interesting!
I would say protocol, block and items are the minimum data to add to create a new version here

do ask if you have any question on how to get those

**Comment by halotroop2288** [comment]
This isn't likely to get updated for a long time, so I'm gonna close this for now. Sorry.

**Pull request closed without merging**